<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPlainTextEdit" name="plainTextEdit_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="plainText">
    <string/>
   </property>
   <property name="placeholderText">
    <string>IP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>清屏</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="port_txt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="plainText">
    <string/>
   </property>
   <property name="placeholderText">
    <string>PORT</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="server_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
